app/src/main/java/ch/hikemate/app/model/route/ListOfHikeRoutesViewModel.kt

```diff
@@ -63,6 +69,33 @@ open class ListOfHikeRoutesViewModel(
     viewModelScope.launch { getRoutesAsync(onSuccess = onSuccess, onFailure = onFailure) }
   }
 
+  private suspend fun getRoutesElevationAsync(
```

Comment:
Question: Why is this elevation feature in the list of hike routes vM? Was ist just ease of use? Because just by name it does not really make sense to have it here, but I also see that there are not really any nice alternatives

Reply:
It's part of the list of hike routes IMO. As you would retrieve their data, you would retrieve their elevation.

Reply:
Alright, after some discussion on the telegram group, lets leave in in, also in the interest of time, however we should consider cleaning up with vM or renaming it. I have created a task in the backlog for this for a future sprint.